code/modules/mob/mob.dm

```diff
@@ -294,6 +294,8 @@
 	set name = "Point To"
 	set category = "Object"
 
+	usr.visible_message("<b>[src]</b> points to [A]")
```

Comment:
Move this to the _pointed so it cant fail

Reply:
to the....what?

Reply:
the proc just below